src/app/(auth)/login/page.tsx

```diff
@@ -45,13 +49,15 @@ export default function Login() {
           <TextInput
             label="Password"
             placeholder="Password"
-            errorText="" // "Password Error"
+            errorText={
+              password === '' || password.length < 6 ? 'Invalid Password' : ''
```

Comment:
(cc @davidqing6432) can we get rid of the `password.length < 6` check here? think that once they're at the point of logging in, we shouldn't be checking any password constraints

Reply:
i can get rid of them!